### PR TITLE
Updated Acknowledgements

### PR DESCRIPTION
**Pull request by johaq**
Updated the list of names in acknowledgements. Moved every current member of a committee up  and former members down into the appropriate sections. 
Please check the spelling of your name.
Please check if I forgot someone.
Please check if everyone is in the right section.

Removed the github handles since I don't have them for everyone. Maybe we should add current affiliation to the name?

### CODE REVIEW

File: pages/acknowledgments.tex [hunk 1]
```diff
@@ -26,62 +26,64 @@ \section*{How to cite this rulebook}
...
+	\begin{minipage}{0.8\textwidth}
+		\begin{multicols}{2}%
+			\footnotesize
+			\noindent%
+			Alex Mitrevski\\
+			Caleb Rascon\\
+			Komei Sugiura\\
+			Mark Finean\\
+			Lars Janssen\\
+			Loy van Beek\\
```
Comment (LoyVanBeek) [inline pages/acknowledgments.tex:78]: Have been TC some years back. I still get the notifications :-)

Comment (johaq) [reply pages/acknowledgments.tex:78]: Well I put you in the "people who contributed to the RoboCup@Home league with their feedback and comments" section. I think that definitely still fits you.

Comment (johaq) [reply pages/acknowledgments.tex:78]: Oh I just realized I totally misread the wording of the first section. I thought it said current members... 
will update

Comment (alex-mitrevski) [reply pages/acknowledgments.tex:78]: IMHO this undervalues the work of previous TC members; being a TC member and actively working on the rulebook is obviously more work than "just" providing feedback and comments. Maybe you could add sections for "current TC members" and "previous TC members" if this distinction is important?

Comment (johaq) [reply pages/acknowledgments.tex:78]: To me it is not important. I would just move all former members back up in the first section.

**Comment by johaq** [comment]
Ok I now have all current and former committee members (to the best of my knowledge) in first section and people who contributed otherwise in second section

File: pages/acknowledgments.tex (hunk 1 above)
Comment (LoyVanBeek) [reply pages/acknowledgments.tex]: Sorry for the nitpick, there's more important stuff indeed. Thnx though

File: pages/acknowledgments.tex [hunk 2]
```diff
@@ -26,62 +26,64 @@ \section*{How to cite this rulebook}
...
-            Raphael Memmesheimer (@airglow)\\
-            Sebastian Meyer zu Borgsen (@semeyerz)\\
-            Syed Ali Raza (@syedaraza)
-        \end{multicols}
-    \end{minipage}
+	\begin{minipage}{0.8\textwidth}
+		\begin{multicols}{2}%
+			\footnotesize
+			\noindent%
+			Mark Finean\\
```
Comment (MatthijsBurgh) [inline pages/acknowledgments.tex:81]: Don't see how this one is alphabetical

File: pages/acknowledgments.tex [hunk 3]
```diff
@@ -26,62 +26,64 @@ \section*{How to cite this rulebook}
...
+			Maxime St-Pierre\\
+			Peter van Dooren\\
+			Raphael Memmesheimer\\
+			Sammy Pfeiffer\\
+			Sebastian Meyer zu Borgsen\\
+			Sven Wachsmuth\\
+			Tijn van der Zant\\
+			Yuma Yoshimoto
+		\end{multicols}
+	\end{minipage}
```
Comment (MatthijsBurgh) [inline pages/acknowledgments.tex:71]: This one isn't alphabetical either

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]
Lists are not alphabetical, while it is stated they are...

File: pages/acknowledgments.tex (hunk 3 above)
Comment (johaq) [reply pages/acknowledgments.tex]: Should be fixed now

**Review by MatthijsBurgh — APPROVED** [review]

**Pull request merged**